sdk/src/connections/target/addi9036_sensor.cpp

```diff
@@ -86,10 +72,11 @@ struct VideoDev {
 };
 
 struct Addi9036Sensor::ImplData {
+    uint8_t numVideoDevs;
```

Comment:
this should be const.

Reply:
It can't be. By default it is set to 1 when ImplData is constructed but then a value gets assigned to it in the open() function of Addi9036Sensor.